tiddlers/usage.tid

```diff
@@ -39,7 +39,3 @@ delimited) checklist items will be displayed as a single checklist.
 List items support any inline formatting rules. This means you can create links,
 bold, or anything else found on the
 [[Formatting in WikiText|http://tiddlywiki.com/static/Formatting%2520in%2520WikiText.html]] guide.
-
```

Comment:
This should be addressed as part of #30. I would like to not remove this right now because it is a good reminder that it needs to be replaced with a link to the actual documentation.